language-support/ts/daml-ledger/index.ts

```diff
@@ -1270,9 +1290,10 @@ class Ledger {
   async archive<T extends object, K, I extends string>(
     template: Template<T, K, I>,
     contractId: ContractId<T>,
+    meta?: CommandMeta,
```

Comment:
Also why would you pass `meta` to archive? Is it to identify the contract itself being archived?

Reply:
Same deal as above, just consistency. Happy to remove if you prefer.